examples/monitoring-alert/README.md

```diff
@@ -0,0 +1,30 @@
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| [terraform](https://www.terraform.io/downloads.html) | >= 0.13 |
+| [google](https://github.com/hashicorp/terraform-provider-google) | ~> 4.0 |
```

Comment:
I would remove the providers section as this is an example.  FYI: The module requires TF v1.3 and TPG >= 4.83.0, < 7.